docs/operations.md

```diff
@@ -0,0 +1,346 @@
+# Operations
+
+This is the work in progress operations guide for showing information,
+which are relevant for production use.
+
+Skipper is proven to scale with number of routes beyond 200.000 routes
```

Comment:
i think we can just drop this sentence. It's so relative.

Reply:
I disagree, because this is one of the measurements we can provide for the audience. Which else project has a number like that?
I think we should show numbers that show the heavy use of our technology.